quetz/cli.py

```diff
@@ -463,7 +482,8 @@ def create(
     config = Config(str(config_file))
 
     deployment_folder.joinpath('channels').mkdir(exist_ok=True)
-    with working_directory(path):
+
+    with working_directory(db_path):
```

Comment:
Could we use the `deployment_folder` as a working directory if it exists?
I'm testing this PR with a local store, so I need to specify a working directory, and instead of using this directory to store the SQLite database, it creates the database relative to CWD. I think it would be better to reuse the `deployment_folder` since we have it, and if there is not a `deployment_folder` then we use CWD.

Reply:
I don't think so, because the `$QUETZ_SQLALCHEMY_DATABASE_URL` includes the relative path to the database. It would be broken if it startd from the deployment_folder.

The best way to keep it generic is to let the user define it in the database URL:

```
export QUETZ_SQLALCHEMY_DATABASE_URL="sqlite:///./deployment_folder/quetz.sqlite"
```

Reply:
Ooh, that's right.